public/stylesheets/cards.css

```diff
@@ -45,10 +45,9 @@
 .card-shadow {
   height: 70vh;
 }
-
 .card-button {
   background-color: rgba(1, 172, 1, 0.801);
-  width: 50%;
+  width: 100%;
```

Comment:
Instead of overriding the style for `play` page, you could use a wrapper that sets `width` to 50%

Reply:
did that, though had to use `width: 100%` to make it look properly when using a wrapper `div`